src/main/kotlin/com/exactpro/th2/lwdataprovider/handlers/AbstractParsedStoredMessageHandler.kt

```diff
@@ -51,8 +51,9 @@ internal abstract class AbstractParsedStoredMessageHandler(
             handler.requestReceived()
         }
         details += detail
+        currentBatchSize += data.serializedSize
         handler.handleNext(detail)
-        if (details.size >= batchSize) {
+        if (details.size >= batchSize || currentBatchSize >= batchSizeBytes) {
```

Comment:
Maybe remove batchSize if `This parameter does not have any effect anymore.`

Reply:
This `batchSize` is not the same batch size now. This parameter limits the batch size to the maximum number of messages per request. We must use it in order to meet the limit requirements. Otherwise, we won't be able to publish a big batch that has more messages than the limit per request